py/picca/fitter2/chi2.py

```diff
@@ -122,12 +145,17 @@ def export(self):
         g.attrs['list of free pars'] = self.best_fit.list_of_vary_param()
         g.attrs['list of fixed pars'] = self.best_fit.list_of_fixed_param()
 
+        ## write down all attributes of the minimum
+        dic_fmin = utils.convert_instance_to_dictionary(self.best_fit.get_fmin())
+        for item, value in dic_fmin.items():
+            g.attrs[item] = value
+
```

Comment:
When you write down all these attibutes, don't we get duplicated info? E.g., fval will be written twice. Could you remove the old instances of the duplicated output parameters?

Reply:
It was not written twice but only once: 1st time written, 2nd overwritten.
Anyway it is more beautiful to write it once.

Reply:
I hadn't realize that, but thanks for caring about the beauty of the code!